src/app/pages/administration/confirm/listing.html

```diff
@@ -1,5 +1,6 @@
 <div class="listing-inspect">
-  <div ng-switch="$ctrl.stage" class="listing-inspect__header">
+  <div ng-switch="$ctrl.stage" class="listing-inspect__header-container">
+    <div class="listing-inspect__header-container">
```

Comment:
I'm leery of these changes as we haven't changed much in these files before. Not completely against it, but want to check on it.

That said, the indentation should have changed. This line added a new level of a `<div>`, which means the following lines should have been indented by another level